examples/acquiring_data/downloading_cutouts.py

```diff
@@ -0,0 +1,93 @@
+"""
+===============================================
+Requesting cutouts of AIA images from the JSOC
+===============================================
+
+This example shows how to request a cutout of a series of
+AIA images from the JSOC and animate the resulting sequence.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.time
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import ImageNormalize, SqrtStretch
+
+import sunpy.map
+from sunpy.net import Fido, attrs, jsoc
+
+#####################################################
+# First, query a full frame AIA image.
+t0 = astropy.time.Time('2012-09-24T14:56:03', scale='utc', format='isot')
+q = Fido.search(
+    attrs.Instrument.aia,
+    attrs.Physobs.intensity,
+    attrs.Wavelength(171*u.angstrom),
+    attrs.Time(t0, t0 + 13*u.s),
+)
+m = sunpy.map.Map(Fido.fetch(q))
+
+#####################################################
+# Next, we will create a submap from this image. We will
+# crop the field of view to active region NOAA 11575.
+m_cutout = m.submap(
+    SkyCoord(-500*u.arcsec, -275*u.arcsec, frame=m.coordinate_frame),
+    top_right=SkyCoord(150*u.arcsec, 375*u.arcsec, frame=m.coordinate_frame),
+)
+m_cutout.peek()
+
+#####################################################
+# We want to watch the evolution of this active region
+# in time, but we do not want to download the full frame
+# image at each timestep. Instead, we will use our submap
+# to create a cutout request from the JSOC.
+#
+# First, create a `~sunpy.net.jsoc.JSOCClient` instance.
+c = jsoc.JSOCClient()
```

Comment:
Why are we not using Fido here?

Reply:
I wanted to separate the request and download steps so I used the client directly. Thinking about it now, there's probably no reason to.

Reply:
If I use `Fido.search` instead of `JSOCClient.search`, I get

```python-traceback
Traceback (most recent call last):
  File "downloading_cutouts.py", line 61, in <module>
    q = Fido.search(
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/net/fido_factory.py", line 275, in search
    results = query_walker.create(query, self)
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/net/attr.py", line 547, in create
    return self.createmm(self, *args, **kwargs)
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/util/functools.py", line 18, in wrapper
    return dispatcher.dispatch(args[1].__class__)(*args, **kwargs)
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/net/fido_factory.py", line 204, in _create_and
    return factory._make_query_to_client(*query.attrs)
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/net/fido_factory.py", line 416, in _make_query_to_client
    candidate_widget_types = self._check_registered_widgets(*query)
  File "/Users/willbarnes/Documents/work/codes/sunpy/sunpy/net/fido_factory.py", line 397, in _check_registered_widgets
    raise NoMatchError("This query was not understood by any clients. Did you miss an OR?")
sunpy.util.datatype_factory_base.NoMatchError: This query was not understood by any clients. Did you miss an OR?
```

when I include the `Cutout` attr.

Reply:
I think you need to add `Cutout` here: https://github.com/sunpy/sunpy/blob/master/sunpy/net/jsoc/jsoc.py#L915